app/src/main/java/com/example/mykotlincalculator21/mvp/view/CalculatorView.kt

```diff
@@ -11,4 +13,25 @@ class CalculatorView(mainActivity: MainActivity, private val binding: ActivityMa
         binding.screenTextExample.setText(number)
     }
 
+    override fun showErrorDivision() {
+        val context = getContext()
+        if (context != null) {
```

Comment:
instead of this if you can use let?.{ .. }

Reply:
Thx for the tip, now its no more